list1.cpp

```diff
@@ -0,0 +1,88 @@
+#include "List1.h"
+
+List1::List1() : mHead(nullptr), mLength(0) {}			//constructor for List1
+
+int List1::length() const {								//implementation of length()
+	return mLength;
+}
+
+void List1::insert(int index, const int value) {		//implementation of insert()
+	mLength++;
+
+	Node * n = new Node;
+	(*n).value = value;
```

Comment:
using the `(*n).value` notation instead of `n->value` is okay here, especially if you think it's more clear -- but eventually you should get used to the `->` syntax